be1-go/hub/election.go

```diff
@@ -87,6 +90,7 @@ func (c *laoChannel) createElection(msg message.Message) error {
 		data.EndTime,
 		false,
 		getAllQuestionsForElectionChannel(data.Questions),
+		c.attendees,
```

Comment:
Are the attendees for an election supposed to change over time with every roll call or should they remain the same as they were at the time the election was created?

Reply:
I think the attendees should remain the same on election creation time, it would be like creating an election for the currently present participants and if someone is part of a roll call after the election has been created it should not be considered as attendee for now

Reply:
I see - maps are passed by reference in Go, so your election's attendees change every time `c.attendees` changes (which it does after every roll call).

I'd suggest creating a custom data type with a Copy method. 

```go
type Attendees struct {
    sync.Mutex,
    store map[string]struct{}
}

func NewAttendees() Attendees {

}

func (a Attendees) IsPresent(key string) ok {

}

func (a Attendees) Add(key string) {

}

func (a Attendees) Copy() Attendees {
     a.Lock()
     defer a.Unlock()

     copy := NewAttendees()

     for key := range a {
         copy.store[key] = struct{}
     }

     return copy
}

Reply:
But I even think that case is acceptable too, arriving at the role call late shouldn't penalize you from voting right? Both scenarios seem actually to make sense, so I am not sure which is better

Reply:
Seems like a question for the evoting channel on slack ;)